lib/src/components/NxColorPicker/NxColorPicker.scss

```diff
@@ -28,6 +28,7 @@
     &:hover {
       .nx-color-picker__color {
         stroke: var(--nx-color-text-stark);
+        stroke-width: var(--nx-border-width-alt);
```

Comment:
If I'm not mistaken, the middle of the stroke width matches the circle's radius, so by increasing the stroke-width by 1px in dark mode, it would also slightly increase the size of the color picker that is being hovered over (you can see that a bit better when the focus line is visible- the separation between the focus outline and hover outline is a tad bit closer). In Sketch it says that the hover border should be an inside border, so you may need to investigate further.

Reply:
Thanks for catching it!
In order to main the circle size, the radius of circle should be changed based on the different stroke width in light and dark mode when hovered, here is the updates f1bbda7c69710944b7f653d3680d315ac345c529.

Reply:
> Thanks for catching it!
In order to main the circle size, the radius of circle should be changed based on the different stroke width in light and dark mode when hovered, here is the updates f1bbda7.

The `r` specified in the JSX presumably can be removed now

Reply:
> The r specified in the JSX presumably can be removed now

As discussed in WW, without further changes the `r` in the JSX cannot in fact be removed because it is needed for the non-hover case.  And we can't just remove `:hover` from the selectors in this PR because the `r` values need to be slightly different in the non-hover case.

However, it still doesn't sit well with me that the `r` is being controlled in two completely different places.  I think a better option would be to just always put a stroke on the `.nx-color-picker__color`, but to have the stroke color match the fill when not hovered.  In fact it looks like this is already what's happening to some extent: there's an `&__color` scss block where this happens but the problem is that it's hard coded to use a stroke-width of 1px; that should probably be changed to `var(--nx-border-width-alt)`.  Then the `r` could also be set there (for both the selected and unselected cases) and the `r` attribute in the JSX could be removed.

Reply:
Agreed, handling the r attributes in one place is much more cleaner.
Here are the updates, ba92021baacf7ed59a5eca404b103059ded0e1bb, thanks.